### ISSUE TITLE
support (IN)VISIBLE index feature

### ISSUE THREAD

**Issue by 0x4261756D** [dolthub/dolt#6690]
There is no support for the `VISIBLE` and `INVISIBLE` keywords for indexes.
This is especially bothering since MySQL Workbench always adds one of those keywords when creating an index via the GUI.
**Steps to reproduce:**
```sql
 CREATE TABLE `dolt_test`.`a` (`id` INT, `b` DOUBLE, PRIMARY KEY (`id`), INDEX `c` (`b` ASC) INVISIBLE);
```
produces
```
syntax error at position 102 near 'INVISIBLE'
CREATE TABLE `dolt_test`.`a` (`id` INT, `b` DOUBLE, PRIMARY KEY (`id`), INDEX `c` (`b` ASC) INVISIBLE)
```

**Comment by fulghum** [comment]
Hey @0x4261756D, thanks for reporting this gap! 🙏  Adding syntax support for this (i.e. parsing, but treating it as a no-op) should be pretty easy. I'm wrapping up some testing on another task, then I'll jump on to this and see if I can get our grammar updated to allow this syntax. Stay tuned!

**Comment by fulghum** [comment]
The changes to the parser to support this syntax were merged in. Once a few more CI steps complete, I'll be able to cut a release and will let you know when the fix is available to pick up.

**Comment by fulghum** [comment]
@0x4261756D – This fix is now available in [Dolt 1.16.2](https://github.com/dolthub/dolt/releases/tag/v1.16.2). 

Thank you for taking the time to file this issue so we could get this fixed! 🙏  Let us know if you run into any other problems with Dolt and we'll be happy to jump on them.
